bigbluebutton-html5/private/config/settings.yml

```
@@ -997,6 +997,7 @@ public:
   whiteboard:
     annotationsQueueProcessInterval: 60
     cursorInterval: 150
+    allowInfiniteCanvas: true
```

```suggestion
    allowInfiniteCanvas: false
```

Note, we should commit this to `false` for now. Administrators should intentionally enable this, knowing it's experimental, there is no recording, etc.